src/service/healthcheck_http_server.cc

```diff
@@ -0,0 +1,86 @@
+#include "healthcheck_http_server.h"
+
+namespace authservice {
+namespace service {
+
+HealthcheckHttpConnection::HealthcheckHttpConnection(
+    HealthcheckAsyncServer& parent,
+    const std::vector<std::unique_ptr<filters::FilterChain>>& chains,
+    tcp::socket sock)
+    : sock_(std::move(sock)), chains_(chains), parent_(parent) {
+  startRead();
+}
+
+void HealthcheckHttpConnection::startRead() {
+  http::async_read(sock_, read_buffer_, request_,
+                   [this](auto, auto) { onReadDone(); });
+}
+
+void HealthcheckHttpConnection::startWrite() {
+  http::async_write(sock_, response_, [this](auto ec, auto) {
+    ec_ = ec;
+    onWriteDone();
+  });
+}
+
+void HealthcheckHttpConnection::onReadDone() {
+  response_.version(request_.version());
+  http::status status = http::status::ok;
+
+  if (request_.method() != http::verb::get || request_.target() != "/healthz") {
+    status = http::status::bad_request;
+  } else {
+    for (auto&& chain : chains_) {
+      if (!chain->jwksActive()) {
+        spdlog::warn("{}: chain:{} is inactive yet", __func__, chain->Name());
```

Comment:
make it clear that chain:{} JWKS is not ready yet? people who read the log will not see the code around here immediately and adding jwks help them for a faster troubleshooting time.